tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -68,6 +68,8 @@ def test_import_edit_export_contact(self):
         self.assertEqual(self.sim_contact['tel'][0]['value'], self.contact.tel['value'])
 
     def tearDown(self):
-        self.data_layer.delete_sim_contact(self.sim_contact['id'])
+        for contact in self.data_layer.sim_contacts:
+            if contact['name'][0].startswith('Name'):
+                self.data_layer.delete_sim_contact(contact['id'])
```

Comment:
That is a good idea. The only problem is that we're already piling up contacts with 'Name***' on them in the SIM cards.